models/aged_creditor.json

```diff
@@ -7,21 +7,21 @@
             "attributes": {
                 "code": {
                     "column": "demarcation_code",
-                    "label": "Code",
+                    "label": "Demarcation Code",
                     "type": "string",
                     "description": "Demarcation Code desc"
                 },
                 "desc": {
-                    "column": "demarcation_desc",
-                    "label": "Description",
+                    "column": "demarcation_label",
+                    "label": "Name",
                     "type": "string",
                     "description": "some description"
```

Comment:
I think this gets redundant when the dimension label is municipality and the attribute label is Name. I think it's best leaving the description blank in such cases